janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -1054,8 +1059,9 @@ public boolean supports(KeyInformation information) {
 
     @Override
     public String mapKey2Field(String key, KeyInformation information) {
-        Preconditions.checkArgument(!StringUtils.containsAny(key,new char[]{' '}),"Invalid key name provided: %s",key);
-        return key;
+        Preconditions.checkArgument(!StringUtils.containsAny(key, new char[]{ REPLACEMENT_CHAR }),
+            "Invalid key name containing reserved character • provided: %s", key);
```

Comment:
This string should use the constant `REPLACEMENT_CHAR`, no?

Same for the other 3 copies of this code.

Reply:
I'm agree